docs/tutorial/index.md

```diff
@@ -1,16 +1,16 @@
 (tutorial)=
 # Get started with Juju
 
-Juju is an application orchestration tool for charmed applications -- that is, applications whose installation, configuration, integration, scaling, upgrading, etc., operations have been defined in a way that Juju can interpret in software packages called 'charms'.
+Juju is a tool for provisioning cloud infrastructure as well as deploying and operating applications on that infrastructure using charms. Charms are software packages that contain instructions for how to operate an application. Juju and charms work together to provide a cloud- and application-agnostic operations solution for any major operation (provision, install, configure, integrate, scale, upgrade, ...) on any type of cloud (Kubernetes or machines).
```

Comment:
> instructions

Not sure about this. This feels cryptic and if we want to not be cryptic we should be more explicit. How about just "code"?

Also there is a trailing ``-` on `cloud-`